src/modules/clearCompleted.js

```diff
@@ -0,0 +1,26 @@
+import { TasksClass } from './tasksClass.js';
+
+// clear all completed
+export const clearComplete = () => {
+  //  get the elements
+  const clearCompletedTask = document.querySelector('#clear-complete');
+  const taskBoxEls = document.querySelectorAll('.task-select-input');
+  const taskStatus = new TasksClass();
+  // get index and checkbox value by iterating
+  taskBoxEls.forEach((theEl) => {
+    // functions to execute when a task checkbox changes
+    const clearCompleted = () => {
+      // clear all check marks
+      theEl.checked = theEl.unchecked;
+      const clearChecks = theEl.checked;
+      // parse the index and false status to array
+      if (clearChecks === false) {
+        taskStatus.clearCompletedTask(clearChecks);
+      }
+    };
+    // events
+    clearCompletedTask.addEventListener('click', clearCompleted, false);
+  });
+};
```

Comment:
Your clear-all completed function doesn't quite work as expected. The expected functionality is to remove all tasks which have the completed property set to true from the array and save the updated array in local storage. Your current implementation simply updates all tasks which have the completed property set to true back to false. Please update your project so the clear all completed button works as expected🙏

Reply:
Thanks. Now fixed.